spot_driver/launch/spot_depth_publishers.launch.py

```diff
@@ -0,0 +1,80 @@
+# Copyright [2023] Boston Dynamics AI Institute, Inc.
+
+
+import launch
+import launch_ros
+import yaml
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+
+
+def launch_depth_register_nodelets(
+    context: launch.LaunchContext,
+    spot_name: LaunchConfiguration,
+    camera_sources_yaml: LaunchConfiguration,
+    ld: launch.LaunchDescription,
+) -> None:
+    composable_node_descriptions = []
+
+    camera_sources_yaml = camera_sources_yaml.perform(context)
+    if not camera_sources_yaml or camera_sources_yaml == "None":
+        camera_sources = ["frontleft", "frontright", "left", "right", "back", "hand"]
+    else:
+        with open(camera_sources_yaml, "r") as yaml_file:
+            camera_sources = yaml.safe_load(yaml_file)["sources"]
+
+    cameras_str = ""
```

Comment:
What's this used for?

Reply:
It was used for debugging, I'll remove this